### ISSUE TITLE
[MU4 Issue] [Accessibility] Unable to continue navigation with arrows after Esc from text editing

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#11214]
**To Reproduce**
1. Navigate with Alt+arrow and select some text
2. Enter editing mode (Alt+Shift+E or Return)
3. Esc to finish editing
4. Try to continue navigation with Alt+arrow - **it doesn't work**

https://user-images.githubusercontent.com/90187801/163594055-3f72cb3c-ddf0-4c0c-86e6-cd8fb94719a8.mp4

**Platform information**
macOS, Windows, Linux

**Comment by DmitryArefiev** [comment]
Fixed in #11214
